app/components/Mods/ModsPage.tsx

```diff
@@ -10,14 +10,21 @@ import {
   installedModList,
   requiredModList,
   notInstalledModList,
+  enabledAlphaModList,
+  installedAlphaModList,
+  requiredAlphaModList,
+  notInstalledAlphaModList,
 } from '../../store';
 import ModsToolbar from './ModsToolbar';
+import { alphaState } from '../../store';
 
 const ModsPage: React.FunctionComponent = () => {
-  const enabledMods = useRecoilValue(enabledModList);
-  const installedMods = useRecoilValue(installedModList);
-  const notInstalledMods = useRecoilValue(notInstalledModList);
-  const requiredMods = useRecoilValue(requiredModList);
+  const alpha = useRecoilValue(alphaState);
+  
+  const enabledMods = useRecoilValue(alpha ? enabledAlphaModList : enabledModList);
```

Comment:
Instead of constantly checking the `alpha` flag every time you need to access these state values, you should instead move the flag check to inside the selector. This is basically what selectors are for.

In this case you'd just use `enabledModList` for both the alpha and non-alpha mods. And then inside the selector you check the `alphaState` and return the correct list accordingly. Do this with all of the lists that depend on whether alpha mode is enabled or not.

Reply:
I'm not able to add that to a selector. It breaks the entire mod manager.
![image](https://user-images.githubusercontent.com/34462599/169756269-74352d76-711f-4fd4-a4cb-3ec8fbc7f512.png)